addons/account_edi/tests/test_edi.py

```diff
@@ -228,57 +230,6 @@ def test_cron_self_trigger(self):
             self.assertEqual(len(capt.records), 2, "Not all records have been processed in this run, the cron should "
                                                    "re-trigger itself to process some more later")
 
-    def test_prepare_edi_vals_to_export(self):
-        """
-            Test _prepare_edi_vals_to_export return values
-            in the proper format
-
-            tag_ids should be a set of id and tags a proper recordset
-        """
-        account_tag = self.env['account.account.tag'].create({
-            "applicability": "taxes",
-            "country_id": self.env.ref('base.us').id,
-            "name": "Test Tag",
-        })
-        tax = self.env['account.tax'].create({
-            "amount": 15,
-            "amount_type": "percent",
-            "description": "15%",
-            "country_id": self.env.ref('base.us').id,
-            "invoice_repartition_line_ids": [
-                Command.create({
-                    "factor_percent": 100,
-                    "repartition_type": "base",
-                    "sequence": 1,
-                }),
-                Command.create({
-                    "factor_percent": 100,
-                    "repartition_type": "tax",
-                    "sequence": 1,
-                    "tag_ids": [Command.link(account_tag.id)]
-                })
-            ],
-            "name": "Test",
-            "tax_exigibility": "on_invoice",
-            "type_tax_use": "sale"
-        })
-
-        invoices = self.env['account.move'].create({
-            'move_type': 'out_invoice',
-            'invoice_date': '2019-01-01',
-            'date': '2019-01-01',
-            'partner_id': self.partner_a.id,
-            'invoice_line_ids': [Command.create({
-                'product_id': self.product_a.id,
-                'tax_ids': [Command.link(tax.id)]
-            })],
-        })
-        vals = invoices._prepare_edi_vals_to_export()
-        self.assertEqual(
-            vals['invoice_line_vals_list'][0]['tax_detail_vals_list'][0]['tag_ids'], set(account_tag.ids))
-        self.assertEqual(
-            vals['invoice_line_vals_list'][0]['tax_detail_vals_list'][0]['tags'].name, "Test Tag")
```

Comment:
@vin-odoo We just remove this test?  (ok, it did not work, but...)

Reply:
I talked with LAS about it last time. He said that testing these values isn't really useful, what's important was to test the flows which are still done.
I'm open to changing this if you think it's important.

Reply:
It is strange that it was added in the forward port https://github.com/odoo/odoo/pull/79439 when it wasn't in the original PR https://github.com/odoo/odoo/pull/78714

I'm also not sure *at all* what the link is with the IMP being done in that commit. And it doesn't seem useful indeed, even more with the fact that it is black-box instead of white-box testing.